src/App/Resources/AppResources.resx

```diff
@@ -1799,4 +1799,133 @@
   <data name="PersonalOwnershipPolicyInEffect" xml:space="preserve">
     <value>An organization policy is affecting your ownership options.</value>
   </data>
+  <data name="Send" xml:space="preserve">
+    <value>Send</value>
```

Comment:
Be sure to add a comment to all of these that feature the name, `"Send"` that it's a brand/product name and should not be translated.

```xml
<comment>'Send' is a noun and the name of a feature called 'Bitwarden Send'. It should not be translated.</comment>
```

Reply:
Good call

Reply:
I used the same comment for any string with 'Send' in it, though a few of them refer to Send in the plural.  Should the comment reflect that, or does it matter?

Reply:
Hopefully it doesn't matter.